tests/test_bibtex_title.py

```diff
@@ -1,7 +1,7 @@
 import betterbib
 
 
-def test():
+def test_translate_title_capitalized():
```

Comment:
Let's replace those many functions with one with `@pytest.mark.parametrize`, taking an input and a reference output.

Reply:
Should the test cases still have a label specifying for which feature they are?

I originally splitted it, because I would always have on failing test there on python3.9 / my setup and after splitting only a part of the tests failed.

I think my system is missing the dictionary of names somehow or running in the wrong locale

Reply:
I'll handle it later.